src/status_im/data_store/realm/schemas/account/v8/browser.cljs

```diff
@@ -0,0 +1,17 @@
+(ns status-im.data-store.realm.schemas.account.v8.browser)
+
+(def schema {:name       :browser
+             :primaryKey :browser-id
+             :properties {:browser-id :string
+                          :name       :string
+                          :timestamp  :int
+                          :dapp?      {:type    :bool
+                                       :default false}
+                          :url        {:type     :string
+                                       :optional true}
+                          :contact    {:type     :string
+                                       :optional true}
+                          :history-index  {:type :int
+                                           :optional true}}
+             :history    {:type     :vector
```

Comment:
@pacamara `:history` should be in `:properties` ?

Reply:
Thanks! Giving it a try...

Reply:
@flexsurfer Done! Also fixed the `spec/def`s for `:history` and `:history-index` as were wrong. But hasn't made a difference, bug (1) still exists...